<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>543</width>
    <height>228</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>543</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ATOM - New Project</string>
  </property>
  <widget class="QLineEdit" name="projectNameTxt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="projectNameLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Poject Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projectLocTxt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browseProjectLocBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="projectLocLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Poject Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>543</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
